# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gkrellm/glibc.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./src/client.c.orig	2008-10-04 04:33:37.000000000 +0400
+++ ./src/client.c	2009-05-21 12:54:22.000000000 +0400
@@ -1620,7 +1620,7 @@
 
 
 static gint
-getline(gint fd, gchar *buf, gint len)
+getline_(gint fd, gchar *buf, gint len)
 	{
 	fd_set			read_fds;
 	struct timeval	tv;
@@ -1714,7 +1714,7 @@
 
 	while (1)
 		{
-		rs=getline(fd, buf, sizeof(buf));
+		rs=getline_(fd, buf, sizeof(buf));
         if(rs<0)
             return FALSE;
 		if (!strcmp(buf, "</gkrellmd_setup>"))
@@ -1732,7 +1732,7 @@
 	table_size = sizeof(update_table) / sizeof(KeyTable);
 	while (1)
 		{
-		rs=getline(fd, buf, sizeof(buf));
+		rs=getline_(fd, buf, sizeof(buf));
         if(rs<0)
             return FALSE;
 		if (!strcmp(buf, "</initial_update>"))

